### PR TITLE
Add support for selecting TLS version v1.2 or v1.3

### PR DESCRIPTION
**Pull request by hselasky**

### CODE REVIEW

File: src/Settings.cpp [hunk 1]
```diff
@@ -123,8 +123,8 @@ const struct option long_options[] =
 {"ipv6_domain",      no_argument, NULL, 'V'},
 {"suggest_win_size", no_argument, NULL, 'W'},
 {"linux-congestion", required_argument, NULL, 'Z'},
-{"tls", no_argument, NULL, 'E'},
-{"ktls", no_argument, NULL, 'K'},
+{"tls",        required_argument, NULL, 'E'},
```
Comment (BorisPis) [inline src/Settings.cpp]: Please use optional_argument to maintain backward comparability

File: src/Settings.cpp [hunk 2]
```diff
@@ -633,7 +627,31 @@ void Settings_Interpret( char option, const char *optarg, thread_Settings *mExtS
             setKTLS( mExtSettings );
             // fall through
         case 'E': // Use SSL
-            setSSL( mExtSettings );
+            if (isSSL( mExtSettings )) {
+                fprintf( stderr, "The -E option should only be specified once (ignored)\n");
+            } else if (strcmp(optarg, "v1.2") == 0) {
+                setSSL12( mExtSettings );
+                SSL_library_init();
```
Comment (BorisPis) [inline src/Settings.cpp]: Let's try to remove duplicate code here and below

**Review by BorisPis — CHANGES_REQUESTED** [review]

File: src/Settings.cpp (hunk 1 above)
Comment (hselasky) [reply src/Settings.cpp]: Fixed.

File: src/Settings.cpp (hunk 2 above)
Comment (hselasky) [reply src/Settings.cpp]: Fixed.

**Comment by hselasky** [comment]
Can this be merged now?

**Review by BorisPis — APPROVED** [review]

**Pull request merged**